docs/api/training-data-importers.rst

```diff
@@ -51,6 +54,88 @@ configuration file:
    importers:
    - name: "RasaFileImporter"
 
+SkillSelector (experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    This feature is currently experimental and might change or be removed in the future.
+    Please share your feedback on it in the `forum <https://forum.rasa.com>`_ to help
+    us making this feature ready for production.
+
+With this importer you can build a contextual AI assistant by combining
+reusable "building blocks" called skills.
```

Comment:
do we really need to call them skills? do they have to have a name? We already introduce a whole bunch of terms in e.g. the dialogue elements, I'd like to avoid introducing yet another term here. Can we just call them projects?

Reply:
Should we also rename the `SkillSelector` or can we keep it that way?

Reply:
yeah let's change the name all the way down

Reply:
then something like `MultiProjectImporter`?

Reply:
renamed it and reorganized test files